source/simulator/solver.cc

```diff
@@ -313,7 +313,8 @@ namespace aspect
                 if (Utilities::MPI::this_mpi_process(src.block(0).get_mpi_communicator()) == 0)
                   AssertThrow (false,
                                ExcMessage (std::string("The iterative (bottom right) solver in BlockSchurPreconditioner::vmult "
-                                                       "did not converge. It reported the following error:\n\n")
+                                                       "did not converge tolerance " + Utilities::to_string(solver_control.tolerance()) +
```

Comment:
I think you should say "...did not converge to a tolerance of " ...

Same below.